e2e/nomostest/nt.go

```diff
@@ -467,7 +467,7 @@ func (nt *NT) printTestLogs() {
 	nt.testLogs(true)
 	nt.T.Log("[CLEANUP] Printing test logs for running pods")
 	for _, ns := range CSNamespaces {
-		nt.testPods(ns)
+		nt.listAndDescribePods(ns)
 	}
 	nt.T.Log("[CLEANUP] Describing not-ready pods")
 	for _, ns := range CSNamespaces {
```

Comment:
Looks like the `describeNotRunningTestPods` is mostly redundant now, since we always describe all pods above

Reply:
done